flepimop/gempyor_pkg/src/gempyor/seeding.py

```diff
@@ -17,9 +17,7 @@
 
 def _DataFrame2NumbaDict(df, amounts, modinf) -> nb.typed.Dict:
     if not df["date"].is_monotonic_increasing:
-        raise ValueError(
-            "_DataFrame2NumbaDict got an unsorted dataframe, exposing itself to non-sense"
-        )
+        raise ValueError("`_DataFrame2NumbaDict` got an unsorted dataframe.")
```

Comment:
Putting the function name in the exception message is redundant, it will appear in the stack trace.
```suggestion
        raise ValueError("The `df` given is not sorted by the 'date' column.")
```